templates/expo-template-bare-minimum/ios/Podfile

```diff
@@ -31,10 +41,8 @@ target 'HelloWorld' do
     :fabric_enabled => flags[:fabric_enabled],
     # An absolute path to your application root.
     :app_path => "#{Pod::Config.instance.installation_root}/..",
-    #
-    # Uncomment to opt-in to using Flipper
     # Note that if you have use_frameworks! enabled, Flipper will not work
-    # :flipper_configuration => flipper_config,
```

Comment:
We can probably remove the `flipper_config` variable on line 16 as well 

https://github.com/expo/expo/blob/6b0153e8d867e87082c5c7b919e3e1a5c61ed9c4/templates/expo-template-bare-minimum/ios/Podfile#L16